docs/cn/Clients-Alluxio-Java.md

```diff
@@ -17,20 +17,37 @@ Alluxio通过文件系统接口提供对数据的访问。Alluxio中的文件提
 
 要使用Java代码获取Alluxio文件系统客户端，请使用：
 
-{% include File-System-API/get-fileSystem.md %}
+```java
+FileSystem fs = FileSystem.Factory.get();
```

Comment:
@Hao-Kailong Do we still need to keep the file `File-System-API/get-fileSystem.md`?

Reply:
We don't have to

Reply:
@yupeng9 So should I delete those files?

Reply:
yes, please remove the orphaned files.